docs/resources/cce_cluster.md

```diff
@@ -120,6 +120,8 @@ versions are available, choose Dashboard > Buy Cluster on the CCE console. Chang
 
 * `highway_subnet_id` - (Optional, String, ForceNew) The ID of the high speed network used to create bare metal nodes. Changing this parameter will create a new cluster resource.
 
+* `kubernetes_svcip_range` - (Optional, String, ForceNew) The range of kubernetes cluster IP. Changing this parameter will create a new cluster resource.
```

Comment:
please ref to the API document: https://support.huaweicloud.com/en-us/api-cce/cce_02_0236.html

Reply:
fixed